src/ray/common/ray_config_def.h

```diff
@@ -198,3 +198,10 @@ RAY_CONFIG(uint32_t, object_store_get_warn_per_num_attempts, 50)
 /// When getting objects from object store, max number of ids to print in the warning
 /// message.
 RAY_CONFIG(uint32_t, object_store_get_max_ids_to_print_in_warning, 20)
+
+/// Allow up to 5 seconds for connecting to GCS Service.
+RAY_CONFIG(int64_t, gcs_service_connect_retries, 50)
+RAY_CONFIG(int64_t, gcs_service_connect_wait_milliseconds, 100)
+
+/// Whether to enable the gcs service.
+RAY_CONFIG(bool, gcs_service_enabled, false)
```

Comment:
move this above the two configurations for gcs service

Reply:
use a env var instead of config now.